databricks/koalas/sql.py

```diff
@@ -0,0 +1,286 @@
+#
+# Copyright (C) 2019 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import _string
+from typing import Dict, Any, Optional
+import inspect
+import pandas as pd
+
+from pyspark.sql import SparkSession, DataFrame as SDataFrame
+
+from databricks import koalas as ks  # For running doctests and reference resolution in PyCharm.
+from databricks.koalas.utils import default_session
+from databricks.koalas.frame import DataFrame
+from databricks.koalas.series import Series
+
+
+__all__ = ["sql"]
+
+from builtins import globals as builtin_globals
+from builtins import locals as builtin_locals
+
+
+def sql(query: str, globals=None, locals=None, **kwargs) -> DataFrame:
```

Comment:
`DataFrame` refers to pyspark.sql.DataFrame here, right? I would've expected the return type to be a Koalas DataFrame though.

Reply:
This is a koalas DataFrame.

Reply:
See the import on line 26

Reply:
My bad. Didn't see that you imported `pyspark.sql.DataFrame` as `SDataFrame`